propolis/src/vmm/hdl.rs

```diff
@@ -256,6 +257,26 @@ impl VmmHdl {
         guard_space.mapping(size, prot, &self.inner, offset as i64)
     }
 
+    /// Tracks dirty pages in the guest's physical address space.
+    ///
+    /// # Arguments:
+    /// - `start_gpa`: The start of the guest physical address range to track.
+    /// Must be page aligned.
+    /// - `bitmap`: A mutable bitmap of dirty pages, one bit per guest PFN
+    /// relative to `start_gpa`.
+    pub fn track_dirty_pages(
+        &self,
+        start_gpa: u64,
+        bitmap: &mut [u8],
+    ) -> Result<()> {
+        let mut tracker = bhyve_api::vm_dirty_tracker {
+            start_gpa,
+            len: bitmap.len(),
```

Comment:
`len` is in bytes of address space, not pages.  This should be multiplied by the `PAGESIZE` constant.

Reply:
Oops. Done. The comment in the kernel needs to change, too.

Reply:
Also updated the kernel in the current illumos CR.